tasks/extract_cldr_data.js

```diff
@@ -12,49 +12,22 @@ module.exports = function (grunt) {
 
   grunt.registerMultiTask('extract_cldr_data', 'Extract CLDR data and transform it for use in JavaScript.', function () {
 
-    var path      = require('path');
-    var serialize = require('serialize-javascript');
-
-    var extract = require('../lib/extract');
+    var extractData = require('formatjs-extract-cldr-data');
+    var path        = require('path');
+    var serialize   = require('serialize-javascript');
 
     // Merge task-specific and/or target-specific options with these defaults.
     var options = this.options({
-      locales: extract.rootLocales(),
-      plurals: false,
-      fields : false,
-      prelude: ''
+      locales       : undefined,
+      pluralRules   : false,
+      relativeFields: false,
+      prelude       : '',
```

Comment:
I've been doing this lately on purpose to avoid refactoring hazards. Since this is possible in ES5+ and that this package only runs in node, I figured it would be good to start using it. In fact trailing commas in function call expressions is on deck for ES7: https://github.com/tc39/ecma262